app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -201,9 +205,15 @@ export const FormAudience = ({
             <Select
               placeholder="All Countries"
```

Comment:
Could we add the caveat as secondary text underneath the `Form.label` (in all scenarios) instead of doing it only when it's `!isDesktop`?

.. I guess that might be a "don't let the user know that a feature doesn't exist" situation, though

Reply:
Totally makes sense 😄